quickstart-templates/workload-identity-creates-workload-identities/README.md

```diff
@@ -0,0 +1,70 @@
+# Create Workload Identity Federation that can create other workload identity federations
+
+## The Idea
+
+Create federated identity credentials (FIC) at appropriate scopes and controls to deploy company and project specific resource. The process is rooted in a Git pull request review process in order to introduce checkpoints, audit trail and repeatability to process. This should restrict the need for manual work requiring high privileges.
+
+## Example scenario
+
+A high-privilege user creates an initial, bootstrap workload identity federation token between Azure and a certain GitHub repository. The workload in this repository is then allowed to access Microsoft Entra to create further workload identity federation tokens and assign them to other repositories.
```

Comment:
I would remove the work token in this sentence. "A high-privileged user creates an initial, bootstrap workload identity federation between Azure and a certain GitHub repository."